tests_integration/test_data/projects/stage_execute/ScriptInPascalCase.sql

```diff
@@ -0,0 +1 @@
+Select 'script in pascal case';
```

Comment:
Can you add the test in the same PR as soluton?

Reply:
I didn't understand, the test is in both PRs

Reply:
So, what's the purpose of this PR?

Reply:
But why not add it in the same pr as the fix for the bug?

Reply:
Because if I add this file in my PR I have to mark the test with @pytests.skip because it is going to fail, and then I will have to create another PR to remove the mark. So, instead I create this PR to add the file to main branch so that the test will pass in my branch and in main branch. In both cases I would have to create two PRs